test/integration/server.h

```diff
@@ -647,6 +650,7 @@ class IntegrationTestServerImpl : public IntegrationTestServer {
 
   // Owned by this class. An owning pointer is not used because the actual allocation is done
   // on a stack in a non-main thread.
+  std::unique_ptr<Thread::SkipAsserts> skip;
```

Comment:
trailing underscore.

Comment on when we will be instantiating one of these? In the PR I can't see what triggers turning on skipAsserts.